### PR TITLE
fixes-birthDate-component: fixes bug with component not return something

### PR DESCRIPTION
**Pull request by paschalidi**

### CODE REVIEW

**Review by herman-rogers — APPROVED** [review]

File: src/BirthDate/BirthDate.jsx [hunk 1]
```diff
@@ -2,12 +2,11 @@ import React from "react";
...
 export default function BirthDate({ date, style }) {
-  if (date)
-    return (
-      <div style={style}>
-        {date.split("-")[2]}.{date.split("-")[1]}.{date.split("-")[0]}
-      </div>
-    );
+  return (
+    <div style={style}>
+      {date.split("-")[2]}.{date.split("-")[1]}.{date.split("-")[0]}
```
Comment (lanwen) [inline src/BirthDate/BirthDate.jsx]: will it panic in case of date is undefined?

Comment (herman-rogers) [reply src/BirthDate/BirthDate.jsx]: It'll complain in the proptypes since it is required.

Comment (lanwen) [reply src/BirthDate/BirthDate.jsx]: but shouldn't we handle it gracefully?

Comment (herman-rogers) [reply src/BirthDate/BirthDate.jsx]: Well we could do it, but I don't think its up to the component creator but rather the people using the component. Other wise all of our components will have {exist ? prop : handle gracefully} which doesn't really scale or work. I don't think its unreasonable to require a user to use a component properly.

**Review by lanwen — APPROVED** [review]

**Pull request merged**